test/common/stats/thread_local_store_test.cc

```diff
@@ -90,6 +91,121 @@ class StatsThreadLocalStoreTest : public testing::Test, public RawStatDataAlloca
   std::unique_ptr<ThreadLocalStoreImpl> store_;
 };
 
+class HistogramTest : public testing::Test, public RawStatDataAllocator {
+public:
+  typedef std::map<std::string, Stats::ParentHistogramSharedPtr> NameHistogramMap;
+  InSequence s;
```

Comment:
Not exactly.  https://github.com/google/googletest/blob/master/googlemock/docs/ForDummies.md

>By creating an object of type InSequence, all expectations in its scope are put into a sequence and have to occur sequentially. Since we are just relying on the constructor and destructor of this object to do the actual work, its name is really irrelevant.